### PR TITLE
feat(workflowv2/plugin): artifactory release

### PR DESCRIPTION
**Pull request by fsamin**

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-cds#20165.0 ✘  
* Build
  * Build and Package All ✘  
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Review by richardlt — APPROVED** [review]

File: contrib/grpcplugins/action/artifactory-release/main.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	}
+
+	artifacts := q.GetOptions()["artifacts"]
+	maturity := q.GetOptions()["maturity"]
+	properties := q.GetOptions()["properties"]
+	releaseNotes := q.GetOptions()["releaseNotes"]
+
+	if err := p.perform(ctx, artifacts, maturity, properties, releaseNotes); err != nil {
+		res.Status = sdk.StatusFail
+		res.Status = err.Error()
```
Comment (sguiheux) [inline contrib/grpcplugins/action/artifactory-release/main.go:50]:
```suggestion
		res.Details = err.Error()
```

**Review by sguiheux — CHANGES_REQUESTED** [review]

File: contrib/grpcplugins/action/artifactory-release/artifactory-release.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: artifactory-release
+type: action
+author: "François SAMIN <francois.samin@corp.ovh.com>"
+description: |
+  Release artifactory package from a repository to another
+inputs:
+  artifacts:
+    type: string
+    description: Artifact name or wilcard pattern.
```
Comment (yesnault) [inline contrib/grpcplugins/action/artifactory-release/artifactory-release.yml:9]:
```suggestion
    description: Artifact name or wildcard pattern.
```

**Review by yesnault — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#20168.0 ✔ 
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20168.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#20171.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Review by yesnault — APPROVED** [review]

**Pull request merged**